src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -691,6 +711,50 @@ SGMatrix<T> matrix_prod(SGMatrix<T>& A, SGMatrix<T>& B,
 }
 
 /**
+ * Performs the operation y = \alpha ax + \beta y
+ * This function multiplies a * x (after transposing a, if needed)
+ * and multiplies the resulting matrix by alpha. It then multiplies vector y by
+ * beta. It stores the sum of these two products in vector y
+ *
+ * @param scaling factor for vector ax
```

Comment:
first word after the `@param` should be always the name of the variable in the function description